python/tests/tornado/test_hyperbahn.py

```diff
@@ -54,11 +54,3 @@ def test_request():
             arg3='boo',
             headers={'as': 'qux'},
         )
-
-
-@pytest.mark.gen_test
-def test_register():
```

Comment:
Can we create a TChannel server in the test that receives the advertise call? I would rather not find about failure in that logic in production.